tests/serverspec/default_spec.rb

```diff
@@ -1,12 +1,24 @@
-require 'spec_helper'
-require 'serverspec'
+require "spec_helper"
+require "serverspec"
+
+release_nickname = ""
+if os[:family] == "ubuntu"
+  case os[:release]
```

Comment:
* "lsb_release -c" should be used to get the code name.
```
def code_name
  Specinfra.backend.run_command("lsb_release -c")
             .stdout.strip.split(" ")[1]
end
```
* it is called 'code name' not 'nickname' and the variable name should be "code_name" or something.